### PR TITLE
Add gitignore to generated project

### LINKED ISSUE #42: Add default .gitignore file when creating a new project
`clio_env` and `.clio-cache` are in the working tree when a new project is created. That should not be the case.

### PR DESCRIPTION
**Pull request by andreyluiz**
Fixes #42.

`node cli new test-project` yields:

Screenshot 2019-12-17 at 15 49 51

.gitignore content is:

```
.clio-cache
clio_env

```

### CODE REVIEW

File: cli/commands/new.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ function createPackage(packageName) {
 
   shell.exec(`echo "'Hello World' -> print" > index.clio`);
 
+  shell.exec(`echo ".clio-cache\nclio_env" > .gitignore`);
```
Comment (garritfra) [inline cli/commands/new.js]: shelljs provides an `echo` method that wraps the command for different operating systems. I notice that I didn't do that for the command above it either. Could you substitute `shell.exec()` with `shell.echo()` in both cases?

Comment (andreyluiz) [reply cli/commands/new.js]: Yes, I'll do it.

Comment (andreyluiz) [reply cli/commands/new.js]: Just something, `shell.echo` only prints to `stdout`. I am not able to point it to a file. But instead, I can try this: `shell.ShellString('something\n').to('output.txt')`.

Comment (garritfra) [reply cli/commands/new.js]: Ah, thats probably why I used exec in the first place!😄 Don't mind my comment. Merged

**Pull request merged**